### DISCUSSION TITLE
Is Stellarium compatible with Windows 11?

### DISCUSSION THREAD

**Discussion by voyager-camera** [Stellarium/stellarium#2177] [Q&A]
Hi everyone

I bought a new Windows 11 Home (21H2,8GB) notebook PC for observation.
I installed Stellarium v0.21.3 but it crashed after running for a while.
The memory usage of Stellarium was 400MB or less when it was first started, but it will gradually increase as the display settings are customized.
Stellarium runs fine on a less powerful Windows 10 PC with less than 500MB of memory used, even with customized display settings.
Last October, Microsoft acknowledged a memory leak issue in Windows 11.
Can you say that Stellarium is currently compatible with Windows 11?
The same phenomenon occurs with older versions of Stellarium installed on Windows 11.

voyager_camera

**Comment by gzotti** [comment]
None of the developers has a Win11 system. My systems are too old (yes, 4-5 years is too old! My oldest running Stellarium nicely is from 2008), and I usually run them until they break or there is a significant reason for an upgrade. Win11 is no significant reason until (probably) 2025. 
That said, you answered the question yourself. If Microsoft has acknowledged a memory leak in Win 11, Win11 is not compatible with any program that triggers the memory leak. This is not fault of the program, but of Windows 11, which should be repaired of course. If some Win11 user can identify a memory leak in Stellarium's sources which maybe triggers exclusively in Win11, we accept hints and bugfixes, of course.

**Comment by voyager-camera** [comment]
Hi, gzotti

Thank you for your answer.
I am a weak user of PCs, so when I need a new PC, I choose a commercially available model with an OS pre-installed.
The choice of Windows 11 has nothing to do with my will, as I'm already in a situation where I can't choose Windows 10.
It seems that it was also a disaster that the PC selected this time is an AMD CPU.
Looking at the Windows events, it seems that System32 \ ucrtbase.dll fails when Stellarium crashes, but this doesn't seem to be the cause of the error.
I asked if there was an example of an effective countermeasure in the same situation.
I think about various measures, but it seems unlikely that my ability will solve it.
You have no choice but to wait for Microsoft's fundamental bug fix or to do a clean install of your OS on Windows 10.

voyager_camera

**Reply by 1234f503** [reply]
Compatible, mine is Windows 11

**Comment by adgorn** [comment]
Working well on my new Win 11 machine.  I've got 40gb ram and Intel I7 which helps I'm sure.  One thing I am seeing is the screen is blinking on/off periodically.

**Comment by pat30170** [comment]
I just downloaded the win64.exe (QT6, then QT5) the installation goes well, but stéllarium does not load, I have the icon that appears and disappears after a few moments....

HP laptop 15S-fq0xxx
Intel Celeron N4120 1.10Ghz
32 GB ram
Windows 11 family 23H2
UHD graphics 600
DiretX 12
OpenGL 4.4
SSD 465GB free

Works perfectly on my Windows10 desktop PC but on the laptop there is nothing to do
Attached is a copy of the application log where the 4 errors are my attempts to launch stellarium.

On the 4 errors everything is identical except the line: "Failing process identifier: 0xODC, 0x2934, 0x1C38, 0x2508", and the time and ID of the report.
![erreur_Stellarium](https://github.com/user-attachments/assets/6bc44ec6-e5e3-46df-994d-2cdc5cefe68f)

**Reply by gzotti** [reply]
Without the Stellarium-written logfile 'log.txt' we cannot even speculate. The N4xxx CPU is not exactly powerful. I don't know its graphics capabilities. The Qt5-based build offers running in ANGLE mode. I would try this next.

**Reply by pat30170** [reply]
Good evening,
Here is (late) the content of the log.txt file :

2024-10-30T16:21:26
Operating System: Windows 11 Version 24H2
Compiled using MSVC 19.29.30154.0
Qt runtime version: 6.5.3
Qt compilation version: 6.5.3
Build ABI: x86_64-little_endian-llp64
Addressing mode: 64-bit
Processor architecture: x86_64
Processor name: Intel(R) Celeron(R) N4120 CPU @ 1.10GHz
Processor maximum speed: 1101 MHz
Processor logical cores: 4

PS : The QT5 version does not work either!

**Comment by Dequinzr** [comment]
Bonjour 
J'utilise Windows 11, le driver OpenGL 4.6, Stellarium 24.3 (la dernière version installée depuis le site de Stellarium) et je guide mon télescope Dobson Goto 254 avec Stellarium (configuré avec le driver ASCOM et son extension Synscan) et mon Gsm via l'appli Synscan Pro pour android, le tout en Wi-fi, et ça fonctionne parfaitement.
